src/left-nav.jsx

```diff
@@ -351,15 +353,17 @@ const LeftNav = React.createClass({
         );
     }
     return (
-      <div>
-        {overlay}
+      <div 
+        className={className}
+        style={style}>
+          {overlay}
         <Paper
           ref="clickAwayableElement"
           zDepth={2}
           rounded={false}
           transitionEnabled={!this.state.swiping}
-          className={className}
-          style={this.mergeStyles(styles.root, openRight && styles.rootWhenOpenRight, style)}>
+          containerClassName={containerClassName}
+          containerStyle={this.mergeStyles(styles.root, openRight && styles.rootWhenOpenRight)}>
```

Comment:
also, don't rename the prop NAME, the names should be className and style. they belong to Paper, they have nothing to do with this :grin: